charts/sonarqube-dce/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # SonarQube Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [10.4.1]
+* Update Chart's version to 10.4.1
+* Fix empty ingress annotations in values
```

Comment:
Could you please move the `Fix empty ingress annotations in values` to the `10.4.0` version?

I failed to notice this before, but according to our release policy, this fix must be under the `10.4.0` version. The same applies to the other changes in this PR.